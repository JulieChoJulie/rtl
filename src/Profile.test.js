import React from 'react';
import { render } from '@testing-library/react';
import Profile from './Profile';

describe('<Profile/>', () => {
   it('matches snapshot', () => {
       const utils = render(<Profile username="juliecho1993" name="julie" />);
       expect(utils.container).toMatchSnapshot();
   });
   it('shows the props correctly', () => {
       const utils = render(<Profile username="juliecho1993" name="julie" />);
       utils.getByText('juliecho1993'); // is there the el with the text, "juliecho1993"
       utils.getByText('(julie)'); // is there the el with the text (julie)
       utils.findAllByText(/ju/);
   })
});
