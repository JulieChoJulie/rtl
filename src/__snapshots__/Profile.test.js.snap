// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile/> matches snapshot 1`] = `
<div>
  <div>
    <b>
      juliecho1993
    </b>
    Â 
    <span>
      (
      julie
      )
    </span>
  </div>
</div>
`;
